src/System.Net.Sockets/src/System/Net/Sockets/SocketTaskExtensions.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.Sockets
     public static class SocketTaskExtensions
     {
         public static Task<Socket> AcceptAsync(this Socket socket) =>
-            socket.AcceptAsync();
+            socket.AcceptAsync((Socket)null);
```

Comment:
Was this just broken previously, or am I missing something?

If it was in fact broken, can we add a test for this method?